moe.karaokes.mugen.yml

```diff
@@ -415,6 +415,23 @@ modules:
             x-checker-data:
               type: git
               tag-pattern: ^(\d{1,2}\.\d{1,2}\.\d{1,4})$
+  - name: ed
```

Comment:
place it just before `bc`, it will be easier to maintain the diff with mpv flatpak